rust/actyx/ax/src/cmd/apps/mod.rs

```diff
@@ -1,15 +1,20 @@
+mod license;
 mod sign;
 
 use crate::cmd::AxCliCommand;
 use futures::Future;
 use structopt::StructOpt;
 
+use license::LicenseOpts;
 pub use sign::{create_signed_app_manifest, SignOpts};
 
 #[derive(StructOpt, Debug)]
 #[structopt(version = env!("AX_CLI_VERSION"))]
 /// manage app manifests
 pub enum AppsOpts {
```

Comment:
I get the name was here before but since this manages app manifests and not apps themselves, I think it makes more sense to have this subcommand be renamed to `manifests`.

Thoughts?

Reply:
This reasoning doesn’t apply to the `apps license` subcommand, though. And the intent of the `apps sign` subcommand is to indeed certify the app — I know that this is technically not possible right now, but it remains the intention.